app/frontend/wallet/cardano-wallet.ts

```diff
@@ -262,15 +232,11 @@ const CardanoWallet = (options) => {
   }
 
   async function getUTxOs(): Promise<Array<UTxO>> {
-    try {
-      const addresses = await myAddresses.discoverAllAddresses()
-      return await blockchainExplorer.fetchUnspentTxOutputs(addresses)
-    } catch (e) {
-      throw NamedError('NetworkError')
-    }
+    const addresses = await myAddresses.discoverAllAddresses()
+    return await blockchainExplorer.fetchUnspentTxOutputs(addresses)
```

Comment:
No need to await on return. It may make sense for readability but I see that there is a return value type annotation already, so it's imho redundant: https://eslint.org/docs/rules/no-return-await